Sources/Logger/Loggers/FileLogger/FileLogger.swift

```diff
@@ -52,36 +59,56 @@ public class FileLogger: Logging {
     }
 
     var currentLogFileUrl: URL {
-        suiteName == nil ?
-            logDirURL.appendingPathComponent("\(currentLogFileNumber)").appendingPathExtension(logFilePathExtension) :
-            logDirURL.appendingPathComponent("extension-\(currentLogFileNumber)").appendingPathExtension(logFilePathExtension)
+        if let appName = appName {
+            return logDirURL
+                .appendingPathComponent("\(appName)-\(currentLogFileNumber)")
+                .appendingPathExtension(logFilePathExtension)
+        } else {
+            return logDirURL
+                .appendingPathComponent("\(currentLogFileNumber)")
+                .appendingPathExtension(logFilePathExtension)
+        }
     }
 
     public var levels: [Level] = [.info]
 
     // MARK: - Initializers
 
+    /// This is public constructor for FileLogger.
+    /// - Parameters:
+    ///   - appName: Name of the App used for appendingPathComponent of logDirURL.
+    ///   - appGroupID: Distinguishes log files according to the app environment (main app, extension 1, extension 2, etc...).
```

Comment:
Ne na tohle to `appGroupID` vubec neni. Na rozlisovani tech app instances je prave `appName`.

Reply:
Opraveno.